src/main.rs

```diff
@@ -192,6 +208,46 @@ struct CheckRelease {
     )]
     release_type: Option<ReleaseType>,
 
+    /// Use only the default and explicitly added features.
+    #[arg(
+        long,
+        help_heading = "Features",
+        conflicts_with = "only_explicit_features"
+    )]
+    default_features: bool,
+
+    /// Use no features except the explicitly mentioned ones.
+    #[arg(long, help_heading = "Features")]
+    only_explicit_features: bool,
+
+    /// Use the named features.
```

Comment:
Perhaps mention that these features are added to the selected feature group (heuristically-selected, default features, or none)?

Reply:
Users are "in a hurry" and in practice won't cross-reference multiple places to determine how things work. Instead, they'll read maybe one place and make a guess about how it works. In this case they'll probably guess wrong — that's what we're trying to avoid here.

Just by looking around on our issue tracker, you can see a number of issues opened about things that already at the time were explained in `--help` text, in the repo FAQ, etc. The users that opened them are perfectly capable of reading, but just took the "path of least resistance" and made a guess based on looking at one thing. We can't change that, so we have to meet users where they are and give them all the contextually-relevant information in the place where they expect to find it i.e. the one place they'll read before making their guess about how things work.

Reply:
Okay. So, something like: `Use the named feature. It is added to the selected feature group (heuristically-selected, default features, or none)`?

My issue with this is that 'feature group' is not defined anywhere, along with the `heuristically-selected` group name.

Reply:
Also: how do I go about describing `--baseline-features` and `--current-features`? Making this description lengthy and repeating it three times seems awkward.

Reply:
I think the actual answer was to tweak the other help text instead, so I've added some suggestions there.

Here's what I'd do here:
```suggestion
    /// Add a feature to the set of features being checked.
    /// The feature will be used in both the baseline and the current version of the crate.
```
and then we can tweak the second sentence for `--baseline-features` and `--current-features` as needed.